core/matrix/coo.cpp

```diff
@@ -61,6 +61,7 @@ GKO_REGISTER_OPERATION(advanced_spmv, coo::advanced_spmv);
 GKO_REGISTER_OPERATION(spmv2, coo::spmv2);
 GKO_REGISTER_OPERATION(advanced_spmv2, coo::advanced_spmv2);
 GKO_REGISTER_OPERATION(convert_row_idxs_to_ptrs, coo::convert_row_idxs_to_ptrs);
```

Comment:
@fritzgoebel Can you actually drop the kernels `convert_row_idxs_to_ptrs` and `convert_row_ptrs_to_idxs` from being publicly available? Since we have the `convert_to_csr` and `convert_to_coo` kernels, which we call instead, we don't need the others to be known in the core, so we should be able to remove them from:
- `core/matrix/{coo,csr}.cpp (the `GKO_REGISTER_OPERATION` and any other mention of them)
- `cuda/matrix/{coo,csr}_kernels.hpp`

I would keep the function inside the omp kernels, even though we say it is not implemented. But these functions are more descriptive what exactly the conversion should do.